src/nbl/CMakeLists.txt

```diff
@@ -559,6 +561,7 @@ endif()
 if (UNIX AND NOT ANDROID AND NOT APPLE)
 	target_include_directories(Nabla PUBLIC 
 		${X11_INCLUDE_DIR} 
+		${X11_xcb_INCLUDE_PATH}
 		X11_Xrandr_INCLUDE_PATH
 		${X11_xf86vmode_INCLUDE_PATH}
```

Comment:
so we need any of the other include paths?

Reply:
Probably not.

Reply:
then remove